doc/api/globals.md

```diff
@@ -629,6 +629,40 @@ logical processors available to the current Node.js instance.
 console.log(`This process is running on ${navigator.hardwareConcurrency}`);
 ```
 
+### `navigator.language`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string}
+
+The `navigator.language` read-only property returns a string representing the
+preferred language of the Node.js instance.
```

Comment:
Is that really what this is? Because I can see that being the case in a browser which follows the settings the end-user has. But in this case, it's just what the bundled ICU has.

Reply:
Is there a way to extract the locales from Intl?